certora/config/LiquidityPoolMoneyFlow.conf

```diff
@@ -5,11 +5,13 @@
         "src/EtherFiNodesManager.sol",
         "src/BNFT.sol",
         "src/TNFT.sol",
+        "src/AuctionManager.sol",
```

Comment:
If we add it to the scene, perhaps it's worth checking that it doesn't involve transfer of funds?